web/checks_api.go

```diff
@@ -1,24 +1,18 @@
 package web
 
 import (
-	"fmt"
 	"net/http"
 
 	"github.com/gin-gonic/gin"
 
-	log "github.com/sirupsen/logrus"
-
-	"github.com/trento-project/trento/internal/cluster"
-	"github.com/trento-project/trento/internal/consul"
-	"github.com/trento-project/trento/internal/hosts"
 	"github.com/trento-project/trento/web/models"
 	"github.com/trento-project/trento/web/services"
 )
 
 type JSONChecksSettings struct {
 	SelectedChecks     []string          `json:"selected_checks" binding:"required"`
 	ConnectionSettings map[string]string `json:"connection_settings" binding:"required"`
-	Hosts              Nodes             `json:"hosts"`
+	Hostnames          []string          `json:"hostnames"`
```

Comment:
Do we need to also `make generate` to have swagger docs updated?

Reply:
yes you right pushing it now